openedx/features/survey_report/management/commands/tests/test_generate_report.py

```diff
@@ -33,7 +33,7 @@ def test_generate_report(self, mock_get_unique_courses_offered, mock_get_recentl
         mock_get_generated_certificates.return_value = 4
         mock_get_course_enrollments.return_value = 5
         out = StringIO()
-        call_command('generate_report', stdout=out)
+        call_command('generate_report', no_send=True, stdout=out)
```

Comment:
why not `send=False`? so [here](https://github.com/openedx/edx-platform/pull/31431/files#diff-e0d4ab6a0a657e4a51e67797f4e09101f13cd8bc2820af933963f9e578ea9e56R39) it's easier to read: `if options['send']` ?

Reply:
Because we want to send by default, so you only need to write "--no-send" if you don't want to send it.

But we could try define a default value to the argument  to use "send" instead of "no-send", so let me know.

Reply:
@mariajgrimaldi do you agree?

Reply:
you're right!